src/js/util/gurbani/get-highlight-indices.ts

```diff
@@ -97,7 +97,7 @@ export const getHighlightIndices = (
     }
 
     case SEARCH_TYPES.ROMANIZED_FIRST_LETTERS_ANYWHERE: {
-      baaniWords = baaniWords.map(w => (w.startsWith('i') ? w.slice(1) : w));
+      // baaniWords = baaniWords.map(w => (w.startsWith('i') ? w.slice(1) : w));
```

Comment:
Do we want to keep this?

Reply:
Just have made it as comment for reference